tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java

```diff
@@ -314,7 +314,7 @@ public void testQueryWithPositionalParameters() {
         catalog.save(Product.of("basketball", 14.88, "TEST-PROD-76", Department.SPORTING_GOODS));
         catalog.save(Product.of("baseball cap", 12.99, "TEST-PROD-77", Department.SPORTING_GOODS, Department.CLOTHING));
 
-        Streamable<Product> found = catalog.findByDepartmentCountAndPriceBelow(2, 100.0);
+        Page<Product> found = catalog.findByDepartmentCountAndPriceBelow(2, 100.0);
```

Comment:
Cannot be page because it has no PageRequest
```suggestion
        List<Product> found = catalog.findByDepartmentCountAndPriceBelow(2, 100.0);
```